Spigot-Server-Patches/0410-Add-village-API.patch

```diff
@@ -0,0 +1,365 @@
+-import net.minecraft.server.ChunkMapDistance;
+-import net.minecraft.server.ChunkStatus;
+-import net.minecraft.server.EntityAreaEffectCloud;
+-import net.minecraft.server.EntityArmorStand;
+-import net.minecraft.server.EntityArrow;
+-import net.minecraft.server.EntityBoat;
+-import net.minecraft.server.EntityEgg;
+-import net.minecraft.server.EntityEnderSignal;
+-import net.minecraft.server.EntityEvokerFangs;
+-import net.minecraft.server.EntityExperienceOrb;
+-import net.minecraft.server.EntityFallingBlock;
+-import net.minecraft.server.EntityFireball;
+-import net.minecraft.server.EntityFireworks;
+-import net.minecraft.server.EntityHanging;
+-import net.minecraft.server.EntityHuman;
+-import net.minecraft.server.EntityInsentient;
+-import net.minecraft.server.EntityItem;
+-import net.minecraft.server.EntityItemFrame;
+-import net.minecraft.server.EntityLeash;
+-import net.minecraft.server.EntityLightning;
+-import net.minecraft.server.EntityMinecartChest;
+-import net.minecraft.server.EntityMinecartCommandBlock;
+-import net.minecraft.server.EntityMinecartFurnace;
+-import net.minecraft.server.EntityMinecartHopper;
+-import net.minecraft.server.EntityMinecartMobSpawner;
+-import net.minecraft.server.EntityMinecartRideable;
+-import net.minecraft.server.EntityMinecartTNT;
+-import net.minecraft.server.EntityPainting;
+-import net.minecraft.server.EntityPotion;
+-import net.minecraft.server.EntitySnowball;
+-import net.minecraft.server.EntityTNTPrimed;
+-import net.minecraft.server.EntityTippedArrow;
+-import net.minecraft.server.EntityTypes;
+-import net.minecraft.server.EntityZombie;
+-import net.minecraft.server.EnumDifficulty;
+-import net.minecraft.server.EnumDirection;
+-import net.minecraft.server.EnumMobSpawn;
+-import net.minecraft.server.ExceptionWorldConflict;
+-import net.minecraft.server.Explosion;
+-import net.minecraft.server.GameRules;
+-import net.minecraft.server.GroupDataEntity;
+-import net.minecraft.server.HeightMap;
+-import net.minecraft.server.IBlockData;
+-import net.minecraft.server.IChunkAccess;
+-import net.minecraft.server.MinecraftKey;
+-import net.minecraft.server.MovingObjectPosition;
+-import net.minecraft.server.PacketPlayOutCustomSoundEffect;
+-import net.minecraft.server.PacketPlayOutUpdateTime;
+-import net.minecraft.server.PacketPlayOutWorldEvent;
+-import net.minecraft.server.PlayerChunk;
+-import net.minecraft.server.ProtoChunkExtension;
+-import net.minecraft.server.RayTrace;
+-import net.minecraft.server.SoundCategory;
+-import net.minecraft.server.Ticket;
+-import net.minecraft.server.TicketType;
+-import net.minecraft.server.Unit;
+-import net.minecraft.server.Vec3D;
+-import net.minecraft.server.WorldGenFeatureEmptyConfiguration;
+-import net.minecraft.server.WorldGenHugeMushroomConfiguration;
+-import net.minecraft.server.WorldGenerator;
+-import net.minecraft.server.WorldNBTStorage;
+-import net.minecraft.server.WorldServer;
++import net.minecraft.server.*;
```

Comment:
revert these import changes

Reply:
Yeah, I keep forgetting to set IJ to not do that. I'll revert once I'm done.